User validation failed: DuplicateUserName;DuplicateEmail.
User validation failed: DuplicateUserName;DuplicateEmail.

User validation failed: DuplicateUserName;DuplicateEmail.
User validation failed: DuplicateUserName;DuplicateEmail.
Application startup exception
User validation failed: DuplicateUserName;DuplicateEmail.
Application startup exception
Application startup exception
Application startup exception
Application startup exception
An unhandled exception has occurred while executing the request.
An exception occurred while calculating the database error page content. Skipping display of the database error page.
An unhandled exception has occurred while executing the request.
An exception occurred while calculating the database error page content. Skipping display of the database error page.
An unhandled exception has occurred while executing the request.
An unhandled exception has occurred while executing the request.
An exception occurred while calculating the database error page content. Skipping display of the database error page.
An exception occurred while calculating the database error page content. Skipping display of the database error page.
An unhandled exception has occurred while executing the request.
An unhandled exception has occurred while executing the request.
An exception occurred while calculating the database error page content. Skipping display of the database error page.
An exception occurred while calculating the database error page content. Skipping display of the database error page.
Application startup exception
An unhandled exception has occurred while executing the request.
An unhandled exception has occurred while executing the request.
An exception occurred while calculating the database error page content. Skipping display of the database error page.
An exception occurred while calculating the database error page content. Skipping display of the database error page.
An unhandled exception has occurred while executing the request.
An unhandled exception has occurred while executing the request.
An exception occurred while calculating the database error page content. Skipping display of the database error page.
An exception occurred while calculating the database error page content. Skipping display of the database error page.
An unhandled exception has occurred while executing the request.
An unhandled exception has occurred while executing the request.
An exception occurred while calculating the database error page content. Skipping display of the database error page.
An exception occurred while calculating the database error page content. Skipping display of the database error page.
An unhandled exception has occurred while executing the request.
An unhandled exception has occurred while executing the request.
An exception occurred while calculating the database error page content. Skipping display of the database error page.
An unhandled exception has occurred while executing the request.
An unhandled exception has occurred while executing the request.
An exception occurred while calculating the database error page content. Skipping display of the database error page.
An exception occurred while calculating the database error page content. Skipping display of the database error page.
Application startup exception
An unhandled exception has occurred while executing the request.
An unhandled exception has occurred while executing the request.
An exception occurred while calculating the database error page content. Skipping display of the database error page.
An exception occurred while calculating the database error page content. Skipping display of the database error page.
An unhandled exception has occurred while executing the request.
An unhandled exception has occurred while executing the request.
An exception occurred while calculating the database error page content. Skipping display of the database error page.
An exception occurred while calculating the database error page content. Skipping display of the database error page.
An error occurred using the connection to database 'mhcdb' on server 'sabaserver.database.windows.net,1433'.
An exception occurred while iterating over the results of a query for context type 'chatbot.Data.DishContext'.
Microsoft.Data.SqlClient.SqlException (0x80131904): Cannot open server 'sabaserver' requested by the login. Client with IP address '85.10.56.4' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect.
   at Microsoft.Data.ProviderBase.DbConnectionPool.CheckPoolBlockingPeriod(Exception e)
   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at Microsoft.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry, SqlConnectionOverrides overrides)
   at Microsoft.Data.SqlClient.SqlConnection.Open(SqlConnectionOverrides overrides)
   at Microsoft.Data.SqlClient.SqlConnection.Open()
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerConnection.OpenDbConnection(Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenInternal(Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.Open(Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReader(RelationalCommandParameterObject parameterObject)
   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.Enumerator.InitializeReader(DbContext _, Boolean result)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.Enumerator.MoveNext()
ClientConnectionId:d8b63c58-2491-4642-8a7d-f269417116ee
Error Number:40615,State:1,Class:14
User validation failed: DuplicateUserName.
User validation failed: DuplicateUserName.
An unhandled exception has occurred while executing the request.
An unhandled exception has occurred while executing the request.
An exception occurred while calculating the database error page content. Skipping display of the database error page.
An exception occurred while calculating the database error page content. Skipping display of the database error page.
User validation failed: DuplicateUserName;DuplicateEmail.
An unhandled exception has occurred while executing the request.
An exception occurred while calculating the database error page content. Skipping display of the database error page.
An unhandled exception has occurred while executing the request.
An exception occurred while calculating the database error page content. Skipping display of the database error page.
Failed executing DbCommand (1,318ms) [Parameters=[@__normalizedUserName_0='?' (Size = 256)], CommandType='Text', CommandTimeout='30']
SELECT TOP(1) [a].[Id], [a].[AccessFailedCount], [a].[ConcurrencyStamp], [a].[Discriminator], [a].[Email], [a].[EmailConfirmed], [a].[LockoutEnabled], [a].[LockoutEnd], [a].[NormalizedEmail], [a].[NormalizedUserName], [a].[PasswordHash], [a].[PhoneNumber], [a].[PhoneNumberConfirmed], [a].[SecurityStamp], [a].[TwoFactorEnabled], [a].[UserName], [a].[ChatIds], [a].[LastValidationCode], [a].[TelegramId]
FROM [AspNetUsers] AS [a]
WHERE ([a].[Discriminator] = N'TelegramUser') AND ([a].[NormalizedUserName] = @__normalizedUserName_0)
An exception occurred while iterating over the results of a query for context type 'chatbot.Data.ApplicationDbContext'.
Microsoft.Data.SqlClient.SqlException (0x80131904): Invalid column name 'Discriminator'.
Invalid column name 'Discriminator'.
Invalid column name 'ChatIds'.
Invalid column name 'LastValidationCode'.
Invalid column name 'TelegramId'.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__169_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.Tasks.Task.<>c.<.cctor>b__277_0(Object obj)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
ClientConnectionId:275e4cd4-5a11-418e-b117-69043260bacb
Error Number:207,State:1,Class:16
Failed executing DbCommand (35,042ms) [Parameters=[@__normalizedUserName_0='?' (Size = 256)], CommandType='Text', CommandTimeout='30']
SELECT TOP(1) [a].[Id], [a].[AccessFailedCount], [a].[ConcurrencyStamp], [a].[Discriminator], [a].[Email], [a].[EmailConfirmed], [a].[LockoutEnabled], [a].[LockoutEnd], [a].[NormalizedEmail], [a].[NormalizedUserName], [a].[PasswordHash], [a].[PhoneNumber], [a].[PhoneNumberConfirmed], [a].[SecurityStamp], [a].[TwoFactorEnabled], [a].[UserName], [a].[ChatIds], [a].[LastValidationCode], [a].[TelegramId]
FROM [AspNetUsers] AS [a]
WHERE ([a].[Discriminator] = N'TelegramUser') AND ([a].[NormalizedUserName] = @__normalizedUserName_0)
An exception occurred while iterating over the results of a query for context type 'chatbot.Data.ApplicationDbContext'.
Microsoft.Data.SqlClient.SqlException (0x80131904): Execution Timeout Expired.  The timeout period elapsed prior to completion of the operation or the server is not responding.
 ---> System.ComponentModel.Win32Exception (258): Unknown error: 258
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__169_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.Tasks.Task.<>c.<.cctor>b__277_0(Object obj)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
ClientConnectionId:adf1e7cf-d697-4152-84c0-d79adc3782b5
Error Number:-2,State:0,Class:11
An unhandled exception has occurred while executing the request.
Failed executing DbCommand (35,091ms) [Parameters=[], CommandType='Text', CommandTimeout='30']
SELECT 1
Failed executing DbCommand (1,089ms) [Parameters=[@__normalizedUserName_0='?' (Size = 256)], CommandType='Text', CommandTimeout='30']
SELECT TOP(1) [a].[Id], [a].[AccessFailedCount], [a].[ConcurrencyStamp], [a].[Discriminator], [a].[Email], [a].[EmailConfirmed], [a].[LockoutEnabled], [a].[LockoutEnd], [a].[NormalizedEmail], [a].[NormalizedUserName], [a].[PasswordHash], [a].[PhoneNumber], [a].[PhoneNumberConfirmed], [a].[SecurityStamp], [a].[TwoFactorEnabled], [a].[UserName], [a].[ChatIds], [a].[LastValidationCode], [a].[TelegramId]
FROM [AspNetUsers] AS [a]
WHERE ([a].[Discriminator] = N'TelegramUser') AND ([a].[NormalizedUserName] = @__normalizedUserName_0)
An exception occurred while iterating over the results of a query for context type 'chatbot.Data.ApplicationDbContext'.
Microsoft.Data.SqlClient.SqlException (0x80131904): Invalid column name 'Discriminator'.
Invalid column name 'Discriminator'.
Invalid column name 'ChatIds'.
Invalid column name 'LastValidationCode'.
Invalid column name 'TelegramId'.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__169_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.Tasks.Task.<>c.<.cctor>b__277_0(Object obj)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
ClientConnectionId:0d79ba1a-a24a-4f59-a9a5-859a85dd2f9a
Error Number:207,State:1,Class:16
An unhandled exception has occurred while executing the request.
Failed executing DbCommand (242ms) [Parameters=[], CommandType='Text', CommandTimeout='30']
CREATE TABLE [Categories] (
    [Id] int NOT NULL IDENTITY,
    [Category_name] nvarchar(max) NULL,
    [Img] nvarchar(max) NULL,
    CONSTRAINT [PK_Categories] PRIMARY KEY ([Id])
);
An error occurred while applying the migrations for 'chatbot.Data.DishContext'. See InnerException for details:
An unhandled exception has occurred while executing the request.
An exception occurred while calculating the database error page content. Skipping display of the database error page.
Failed executing DbCommand (217ms) [Parameters=[@__normalizedUserName_0='?' (Size = 256)], CommandType='Text', CommandTimeout='30']
SELECT TOP(1) [a].[Id], [a].[AccessFailedCount], [a].[ConcurrencyStamp], [a].[Discriminator], [a].[Email], [a].[EmailConfirmed], [a].[LockoutEnabled], [a].[LockoutEnd], [a].[NormalizedEmail], [a].[NormalizedUserName], [a].[PasswordHash], [a].[PhoneNumber], [a].[PhoneNumberConfirmed], [a].[SecurityStamp], [a].[TwoFactorEnabled], [a].[UserName], [a].[ChatIds], [a].[LastValidationCode], [a].[TelegramId]
FROM [AspNetUsers] AS [a]
WHERE ([a].[Discriminator] = N'TelegramUser') AND ([a].[NormalizedUserName] = @__normalizedUserName_0)
An exception occurred while iterating over the results of a query for context type 'chatbot.Data.ApplicationDbContext'.
Microsoft.Data.SqlClient.SqlException (0x80131904): Invalid column name 'Discriminator'.
Invalid column name 'Discriminator'.
Invalid column name 'ChatIds'.
Invalid column name 'LastValidationCode'.
Invalid column name 'TelegramId'.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__169_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.Tasks.Task.<>c.<.cctor>b__277_0(Object obj)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
ClientConnectionId:49c6aa4f-d66f-4615-8000-c4b363ba8856
Error Number:207,State:1,Class:16
An unhandled exception has occurred while executing the request.
User validation failed: DuplicateUserName;DuplicateEmail.
User validation failed: DuplicateUserName;DuplicateEmail.
Invalid password for user.
User failed to provide the correct password.
User validation failed: DuplicateUserName;DuplicateEmail.
An unhandled exception has occurred while executing the request.
An exception occurred while calculating the database error page content. Skipping display of the database error page.
An unhandled exception has occurred while executing the request.
An exception occurred while calculating the database error page content. Skipping display of the database error page.
User validation failed: DuplicateUserName;DuplicateEmail.
An unhandled exception has occurred while executing the request.
An exception occurred while calculating the database error page content. Skipping display of the database error page.
User validation failed: DuplicateUserName;DuplicateEmail.
User validation failed: DuplicateUserName;DuplicateEmail.
An unhandled exception has occurred while executing the request.
An exception occurred while calculating the database error page content. Skipping display of the database error page.
User validation failed: DuplicateUserName;DuplicateEmail.
An unhandled exception has occurred while executing the request.
An exception occurred while calculating the database error page content. Skipping display of the database error page.
An unhandled exception has occurred while executing the request.
An exception occurred while calculating the database error page content. Skipping display of the database error page.
An unhandled exception has occurred while executing the request.
User validation failed: DuplicateUserName;DuplicateEmail.
User validation failed: DuplicateUserName;DuplicateEmail.
User validation failed: DuplicateUserName;DuplicateEmail.
User validation failed: DuplicateUserName;DuplicateEmail.
An error occurred using the connection to database 'mhcdb' on server 'sabaserver.database.windows.net,1433'.
An exception occurred while iterating over the results of a query for context type 'chatbot.Data.DishContext'.
Microsoft.Data.SqlClient.SqlException (0x80131904): Cannot open server 'sabaserver' requested by the login. Client with IP address '46.53.251.114' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect.
   at Microsoft.Data.ProviderBase.DbConnectionPool.CheckPoolBlockingPeriod(Exception e)
   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at Microsoft.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry, SqlConnectionOverrides overrides)
   at Microsoft.Data.SqlClient.SqlConnection.Open(SqlConnectionOverrides overrides)
   at Microsoft.Data.SqlClient.SqlConnection.Open()
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerConnection.OpenDbConnection(Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenInternal(Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.Open(Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReader(RelationalCommandParameterObject parameterObject)
   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.Enumerator.InitializeReader(DbContext _, Boolean result)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.Enumerator.MoveNext()
ClientConnectionId:a9af2c31-f3e8-49cf-b088-26a79f24e506
Error Number:40615,State:1,Class:14
User validation failed: DuplicateUserName;DuplicateEmail.
User validation failed: DuplicateUserName;DuplicateEmail.
